gui/wxpython/datacatalog/tree.py

```diff
@@ -489,7 +489,9 @@ def _reloadGrassDBNode(self, grassdb_node):
 
     def _reloadTreeItems(self):
         """Updates grass databases, locations, mapsets and layers in the tree.
-        Saves resulting data and error."""
+        Saves resulting data and error. It runs in thread, so it should not
+        directly interact with GUI.
+        """
```

Comment:
Please, document that this returns errors, i.e., result evaluates to False on success and to True on failure (the opposite of what is usually expected in Python). I assume raising exception is not advantageous here. (?)